qa-tests-backend/scripts/run-compatibility.sh

```diff
@@ -62,15 +64,15 @@ compatibility_test() {
 
     update_junit_prefix_with_sensor_version
 
-    store_qa_test_results "compatibility-test-sensor-${SENSOR_CHART_VERSION}"
-    [[ ! -f FAIL ]] || die "compatibility-test-sensor-${SENSOR_CHART_VERSION}"
+    store_qa_test_results "compatibility-central-v${CENTRAL_CHART_VERSION}-test-sensor-v${SENSOR_CHART_VERSION}"
+    [[ ! -f FAIL ]] || die "compatibility-central-v${CENTRAL_CHART_VERSION}-test-sensor-v${SENSOR_CHART_VERSION}"
 }
 
 update_junit_prefix_with_sensor_version() {
```

Comment:
[nit] Looks like the function should be renamed.

Reply:
Done